filebot-node.json

```
@@ -1,14 +1,14 @@
 {
     "FileBot-Node": {
-        "description": "FileBot Node is a client-server application that'll allow you to run filebot commands.<p>Based on the official FileBot docker image: <a href='https://hub.docker.com/r/rednoah/filebot' target='_blank'>https://hub.docker.com/r/rednoah/filebot</a>.",
+        "description": "FileBot - The ultimate TV and Movie Renamer<p>FileBot is the ultimate tool for organizing and renaming your Movies, TV Shows and Anime as well as fetching subtitles and artwork. It's smart and just works.</p>",
```

re-include statement and link on docker image used:
```suggestion
        "description": "FileBot - The ultimate TV and Movie Renamer<p>FileBot is the ultimate tool for organizing and renaming your Movies, TV Shows and Anime as well as fetching subtitles and artwork. It's smart and just works.</p><p>Based on the official FileBot docker image: <a href='https://hub.docker.com/r/rednoah/filebot' target='_blank'>https://hub.docker.com/r/rednoah/filebot</a>.",
```